server-plugins/chunter-resources/package.json

```diff
@@ -36,6 +37,11 @@
     "@hcengineering/workbench": "^0.6.8",
     "@hcengineering/notification": "^0.6.14",
     "@hcengineering/server-notification": "^0.6.1",
-    "@hcengineering/server-notification-resources": "^0.6.0"
+    "@hcengineering/server-notification-resources": "^0.6.0",
+    "@hcengineering/server": "^0.6.0",
+    "@hcengineering/tiptap": "^0.6.0",
```

Comment:
I expect only our text plugin to be here, please remove tiptap direct dependencies.

Reply:
Removed tiptap dependencies